src/middlewared/middlewared/plugins/vm/devices/storage_devices.py

```diff
@@ -61,6 +62,7 @@ class RAW(StorageDevice):
         Int('size', default=None, null=True),
         Int('logical_sectorsize', enum=[None, 512, 4096], default=None, null=True),
         Int('physical_sectorsize', enum=[None, 512, 4096], default=None, null=True),
+        Str('iotype', enum=['native', 'threads', 'io_uring'], default='native'),
```

Comment:
Our enums are uppercase

Reply:
I'll fix. Is is okay to just .lower() it for the XML (that is why they were lower case here - these are the strings that libvirt recognizes), or would you prefer I assume there is no straight mapping from the XML to the enum and add a mapping as well?

Reply:
Just `.lower()` will be enough, we do it in a lot of similar places

Reply:
SGTM